src/components/SingleEvent/Participant.js

```diff
@@ -28,9 +30,10 @@ const ParticipantWrapper = styled('div')`
   align-items: center;
 `
 
-const TwitterAvatar = styled('img')`
+const Avatar = styled(DefaultAvatar)`
```

Comment:
Better to use `css` function of `react-emotion` and then pass through as a `className` -> https://emotion.sh/docs/css

Reply:
Actually passing through className is standard practice with emotion styled components. As long as you are not actually writing the classnames yourself. You're just exposing the className so emotion can give it dynamic classNames.

https://emotion.sh/docs/styled#styling-any-component